hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutFromLoadAllOperation.java

```diff
@@ -96,7 +104,7 @@ private void addInvalidation(Data key) {
         }
 
         if (invalidationKeys == null) {
-            invalidationKeys = new ArrayList<>(keyValueSequence.size() / 2);
+            invalidationKeys = new ArrayList<>(loadingSequence.size() / 2);
```

Comment:
Minor: this is probably very low importance, but shouldn't the initial capacity take into account the presence of `expirationTime` items?